test/tests/functional/pbs_init_script.py

```diff
@@ -43,23 +43,28 @@ class TestPbsInitScript(TestFunctional):
     """
     Testing PBS Pro init script
     """
+
     def test_env_vars_precede_pbs_conf_file(self):
         """
         Test PBS_START environment variables overrides values in pbs.conf file
         """
-        self.du.run_cmd(cmd=['/etc/init.d/pbs', 'stop'])
-
         conf = {'PBS_START_SERVER': '1', 'PBS_START_SCHED': '1',
-                'PBS_START_COMM': '1', 'PBS_START_MOM': '0'}
+                'PBS_START_COMM': '1', 'PBS_START_MOM': '1'}
+        self.du.set_pbs_config(confs=conf)
+
+        self.du.run_cmd(cmd=['/etc/init.d/pbs', 'stop'], sudo=True)
+
+        conf['START_PBS_MOM'] = '0'
         self.du.set_pbs_config(confs=conf)
 
-        env = {'PBS_START_SERVER': '0', 'PBS_START_SCHED': '0',
-               'PBS_START_COMM': '0', 'PBS_START_MOM': '1'}
+        cmd = ['PBS_START_SERVER=0', 'PBS_START_SCHED=0',
+               'PBS_START_COMM=0', 'PBS_START_MOM=1',
+               '/etc/init.d/pbs', 'start']
```

Comment:
Please use os.sep and os.path.join() to create the path

Reply:
Good point.  I'll update this to use os.path.join()